### ISSUE TITLE
Texture Bugging Out After Transition Between Rooms

### ISSUE THREAD

**Issue by Pokedude1110** [elishacloud/Silent-Hill-2-Enhancements#726]
Randomly after I leave a room, the game will get stuck on one frame when transitioning between areas. When I enter the next room, James' textures bug out rapidly. I don't know why it happens, but it doesn't go away and it usually leads to a crash or freeze. Included here is the log file if that helps at all.
[d3d8.log](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/10830392/d3d8.log)

**Comment by Pokedude1110** [comment]
Also, that frame will repeat every time a room transition happens. It also apparently starts when I leave menus as well.

**Comment by Polymega** [comment]
This is the first report we've had of this issue. Can you tell me where you downloaded the base (vanilla) game from?

I see these reports below in your log file. If you open `SH2EEconfig.exe` and disable "Run the game in DirectX 9" (found on the main screen), does it fix the issue?

```log
2188 00:47:12.554 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:47:12.554 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:47:12.557 Successfully loaded MinimalColorGrading.fx
2188 00:47:12.559 Successfully loaded LiftGammaGain.fx
2188 00:47:12.559 Enabling technique: MinimalColorGrading
2188 00:47:12.559 Enabling technique: LiftGammaGain
2188 00:47:12.561 Successfully loaded MinimalColorGrading.fx
2188 00:47:12.563 Successfully loaded LiftGammaGain.fx
2188 00:47:12.563 Enabling technique: MinimalColorGrading
2188 00:47:12.563 Enabling technique: LiftGammaGain
2188 00:47:12.563 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:47:12.580 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:47:12.606 Loading image files for textures ...
2188 00:47:20.099 m_IDirect3DDevice8::FakeGetFrontBuffer Using GDI to get front buffer data.
2188 00:47:20.982 GetDiskSpace Available disk space smaller than 2TBs: 139941576
2188 00:48:36.281 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:36.281 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:36.285 Successfully loaded MinimalColorGrading.fx
2188 00:48:36.287 Successfully loaded LiftGammaGain.fx
2188 00:48:36.287 Enabling technique: MinimalColorGrading
2188 00:48:36.287 Enabling technique: LiftGammaGain
2188 00:48:36.292 Successfully loaded MinimalColorGrading.fx
2188 00:48:36.295 Successfully loaded LiftGammaGain.fx
2188 00:48:36.296 Enabling technique: MinimalColorGrading
2188 00:48:36.296 Enabling technique: LiftGammaGain
2188 00:48:36.296 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:36.319 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:36.350 Loading image files for textures ...
2188 00:48:38.089 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:38.089 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:38.092 Successfully loaded MinimalColorGrading.fx
2188 00:48:38.095 Successfully loaded LiftGammaGain.fx
2188 00:48:38.095 Enabling technique: MinimalColorGrading
2188 00:48:38.095 Enabling technique: LiftGammaGain
2188 00:48:38.101 Successfully loaded MinimalColorGrading.fx
2188 00:48:38.103 Successfully loaded LiftGammaGain.fx
2188 00:48:38.103 Enabling technique: MinimalColorGrading
2188 00:48:38.103 Enabling technique: LiftGammaGain
2188 00:48:38.103 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:38.135 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:38.166 Loading image files for textures ...
2188 00:48:39.338 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:39.338 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:39.341 Successfully loaded MinimalColorGrading.fx
2188 00:48:39.344 Successfully loaded LiftGammaGain.fx
2188 00:48:39.344 Enabling technique: MinimalColorGrading
2188 00:48:39.344 Enabling technique: LiftGammaGain
2188 00:48:39.352 Successfully loaded MinimalColorGrading.fx
2188 00:48:39.354 Successfully loaded LiftGammaGain.fx
2188 00:48:39.354 Enabling technique: MinimalColorGrading
2188 00:48:39.354 Enabling technique: LiftGammaGain
2188 00:48:39.354 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:39.387 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:39.424 Loading image files for textures ...
2188 00:48:39.632 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:39.633 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:39.639 Successfully loaded MinimalColorGrading.fx
2188 00:48:39.642 Successfully loaded LiftGammaGain.fx
2188 00:48:39.643 Enabling technique: MinimalColorGrading
2188 00:48:39.643 Enabling technique: LiftGammaGain
2188 00:48:39.650 Successfully loaded MinimalColorGrading.fx
2188 00:48:39.653 Successfully loaded LiftGammaGain.fx
2188 00:48:39.654 Enabling technique: MinimalColorGrading
2188 00:48:39.654 Enabling technique: LiftGammaGain
2188 00:48:39.654 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:39.692 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:39.740 Loading image files for textures ...
2188 00:48:40.751 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:40.751 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:40.758 Successfully loaded MinimalColorGrading.fx
2188 00:48:40.760 Successfully loaded LiftGammaGain.fx
2188 00:48:40.761 Enabling technique: MinimalColorGrading
2188 00:48:40.761 Enabling technique: LiftGammaGain
2188 00:48:40.768 Successfully loaded MinimalColorGrading.fx
2188 00:48:40.771 Successfully loaded LiftGammaGain.fx
2188 00:48:40.771 Enabling technique: MinimalColorGrading
2188 00:48:40.771 Enabling technique: LiftGammaGain
2188 00:48:40.772 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:40.805 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:40.841 Loading image files for textures ...
2188 00:48:41.067 Destroyed runtime environment on runtime 1ABFC2B8.
2188 00:48:41.067 Recreated runtime environment on runtime 1ABFC2B8.
2188 00:48:41.073 Successfully loaded MinimalColorGrading.fx
2188 00:48:41.078 Successfully loaded LiftGammaGain.fx
2188 00:48:41.079 Enabling technique: MinimalColorGrading
2188 00:48:41.079 Enabling technique: LiftGammaGain
2188 00:48:41.086 Successfully loaded MinimalColorGrading.fx
2188 00:48:41.090 Successfully loaded LiftGammaGain.fx
2188 00:48:41.090 Enabling technique: MinimalColorGrading
2188 00:48:41.090 Enabling technique: LiftGammaGain
2188 00:48:41.090 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:41.120 m_IDirect3DDevice9::Present Skipping frame after gamma change!
2188 00:48:41.153 Loading image files for textures ...
2188 00:50:04.820 m_IDirect3DDevice8::CreateImageSurface Error: Failed to create image surface! 3840x2160
```
